chainer/functions/__init__.py

```diff
@@ -194,6 +194,9 @@
 from chainer.functions.math.exponential import Log2  # NOQA
 from chainer.functions.math.exponential_m1 import expm1  # NOQA
 from chainer.functions.math.exponential_m1 import Expm1  # NOQA
+from chainer.functions.math.fft import fft  # NOQA
+from chainer.functions.math.fft import FFT  # NOQA
```

Comment:
Is this needed?

Reply:
It looks other FunctionNode classes are defined in this file. Is it better to remove these classes?

Reply:
Yes. Existing definitions are for backward compatibility.